DOCS-761: Add RPlidar mount direction guidance (#1571)
viamrobotics/docs: docs/services/slam/cartographer/_index.md

## Troubleshooting

### Mount an RPlidar to the rover

If you have a Viam Rover and are mounting an RPlidar to your rover, be sure to position the RPlidar so that it faces forward in the direction of travel, facing in the same direction as the included webcam.

> For example, if you are using the RPlidar A1 model, mount it to the Rover so that the pointed end of the RPlidar mount housing points in the direction of the front of the Rover.

Reviewer: Again - let's add a link for the A1 model

This ensures that the generated [SLAM](/services/slam/) map is oriented in the expected direction relative to the Rover, with the top of the generated map corresponding to the direction the RPlidar is facing when you initiate mapping.

If you need a mount plate for your RPlidar, you can 3D print an adapter plate.

> The STL file for an adapter plate is available on [GitHub](https://github.com/viamrobotics/Rover-VR1/blob/master/CAD/RPIidar_adapter.STL).

Reviewer 2: Change link to https://github.com/viamrobotics/Rover-VR1/blob/master/CAD/RPIidarA1_adapter.STL

Add RPlidar A3 link as well: https://github.com/viamrobotics/Rover-VR1/blob/master/CAD/RPIidarA1_adapter.STL

Wait to merge until https://github.com/viamrobotics/Rover-VR1/pull/3 is merged.

> If you need a mount plate for your RPlidar, you can 3D print an adapter plate.
> The STL file for an adapter plate is available on [GitHub](https://github.com/viamrobotics/Rover-VR1/blob/master/CAD/RPIidar_adapter.STL).

Reviewer: ```suggestion
If you need a mount plate for your RPlidar, you can 3D print [this adapter plate](https://github.com/viamrobotics/Rover-VR1/blob/master/CAD/RPIidar_adapter.STL).
```
